index.html

```diff
@@ -1,12 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="css/main.css">
-    <title>Document</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <title>HW - Our Team</title>
 </head>
+
 <body>
-    
+
+  <header class="header">
+    <p class="header__subtitle">The geniuses behind our work</p>
+    <h1 class="header__title">Our team</h1>
+    <div class="divider header__divider">
+      <img class="divider__icon" src="images/icon.svg" alt="icon">
+    </div>
+  </header>
+
+  <main class="main">
+    <section class="hero">
+      <div class="hero__photo">
+        <div class="divider--top hero__divider"></div>
+        <img src="images/photo.png" alt="photo">
+        <div class="divider--bottom hero__divider"></div>
+      </div>
+
+      <div class="divider--vertical hero__divider"></div>
```

Comment:
```suggestion
      <div class="divider divider--vertical hero__divider"></div>
```
Модифікатор не може існувати без Блока